tests/foreman/cli/test_katello_agent.py

```diff
@@ -16,99 +16,89 @@
 
 :Upstream: No
 """
-import time
-
 import pytest
-from broker import VMBroker
 
-from robottelo.api.utils import wait_for_errata_applicability_task
-from robottelo.cli.activationkey import ActivationKey
-from robottelo.cli.factory import make_activation_key
-from robottelo.cli.factory import make_host_collection
+from robottelo import constants
 from robottelo.cli.factory import setup_org_for_a_custom_repo
 from robottelo.cli.factory import setup_org_for_a_rh_repo
-from robottelo.cli.host import Host
-from robottelo.cli.hostcollection import HostCollection
 from robottelo.config import settings
-from robottelo.constants import FAKE_0_CUSTOM_PACKAGE_GROUP
-from robottelo.constants import FAKE_0_CUSTOM_PACKAGE_GROUP_NAME
-from robottelo.constants import FAKE_0_CUSTOM_PACKAGE_NAME
-from robottelo.constants import FAKE_1_CUSTOM_PACKAGE
-from robottelo.constants import FAKE_1_CUSTOM_PACKAGE_NAME
-from robottelo.constants import FAKE_2_CUSTOM_PACKAGE
-from robottelo.constants import FAKE_2_CUSTOM_PACKAGE_NAME
-from robottelo.constants import PRDS
-from robottelo.constants import REPOS
-from robottelo.constants import REPOSET
-from robottelo.hosts import ContentHost
+from robottelo.helpers import InstallerCommand
+
+
+pytestmark = [
+    pytest.mark.run_in_one_thread,
+    pytest.mark.skip_if_not_set('clients', 'fake_manifest'),
+]
+
 
+@pytest.fixture(scope='module')
+def sat_with_katello_agent(module_destructive_sat):
+    """Enable katello-agent on destructive_sat"""
+    module_destructive_sat.register_to_dogfood()
+    # Enable katello-agent from satellite-installer
+    installer_obj = InstallerCommand('foreman-proxy-content-enable-katello-agent true')
+    result = module_destructive_sat.install(installer_obj)
```

Comment:
```suggestion
    result = module_destructive_sat.install(InstallerCommand('foreman-proxy-content-enable-katello-agent true'))
```

Non blocking, code saving comment !

Reply:
Updated